docs/metecho-how-to.md

```diff
@@ -0,0 +1,219 @@
+  Nonprofit Success Pack) on GitHub and ensure that you and anyone you’d like to
+  work with are collaborators.
+
+If you want to create a project, skip down to [Create a
+Project](#create-a-project).
+
+## Step 1 - Select a Project
+**Project** - *(e.g. Nonprofit Success Pack) Projects contain all the work being
+done. They are equivalent to GitHub repositories.*
+
+- Select the Project you want to contribute to.
+- If you do not see the Project you want to work on:
+  - Confirm that you are logged into the correct Metecho account.
+  - Contact an admin for the repository on GitHub and ensure that you are a
+    collaborator.
+  - Select Re-sync Projects in Metecho.
+
+To continue, go to **Step 2 - Create a Task** or, if you have a set of several
+closely related Tasks in mind, skip to [Step 13 - Create an
+Epic](#step-13---create-an-epic)
+
+## Step 2 - Create a Task
+**Task** - *Tasks represent small changes to the Project, and may stand alone or
+be part of an Epic. Creating a Task creates a branch in GitHub.*
+
+- Select the Tasks tab.
+- Create a new Task of your own (or from an existing GitHub Issue).
+  - Name the Task.
+  - The name can be a brief description of the work.
+  - Use the optional description section for more detail, if needed.
+  - Use the recommended dev org type unless you know of a specific reason to use
+    a different one.
+  - Select Create to save and navigate to your Task detail view. Select Create &
+    New to continue creating multiple Tasks that are unrelated to each other.
+- If you need to edit the Task name or description, or delete the Task, select
+  the gear icon in the top right corner. Deleting a Task deletes all the Orgs as
+  well.
+
+## Step 3 - Assign Task Developer
+**Developer** - *The person assigned to do the work of a Task.*
+
+- Assign yourself or another person to be the Developer for each Task you
+  created.
+- If you do not see the person you want to assign, contact an admin for the
+  repository on GitHub and ensure that the person is a collaborator. Then
+  re-sync GitHub collaborators in Metecho.
+- If you need to change or remove the Task Developer, use the drop down menu on
+  the Developer card in the Task detail view.
+
+## Step 4 - Create Dev Org
+**Dev Org** - *A temporary Salesforce org where a Developer can work on
+contributions to a Project. Dev Orgs expire after 30 days, and all unretrieved
+work is deleted.*
+
+- Review the Next Steps for This Task list to see progress and who is
+  responsible for the next step.
+- To begin work, the Developer will need to select Create Org in the Dev Org
+  card.
+- Dev Org creation can take a number of minutes. Feel free to leave the page.
+  Metecho will provide an alert when the Dev Org is ready.
+
+## Step 5 - Make Changes in Dev Org
+- **This step takes place in the new Salesforce Dev Org, not in Metecho.**
+- Select View Org on the Dev Org card to navigate out of Metecho and make your
+  changes in the temporary Salesforce Dev Org.
```

Comment:
Could we add some information about what changes can be made to a dev org?

Reply:
That's a great idea, but I'm pretty sure any text for that would need to come from someone at SFDO -- I personally at least don't have a great sense for realistic changes that users might be making. :) The only examples I've seen are adding/editing "fields and relationships" in the Orgs.